source/intro.tex

```diff
@@ -424,8 +424,8 @@
 
 \pnum
 Throughout this International Standard, each example is introduced by
-``\begin{example}'' and terminated by ``\end{example}''. Each note is
-introduced by ``\begin{note}'' and terminated by ``\end{note}''. Examples
+``[\,\textit{Example:}'' and terminated by ``\textit{\,---\,end example}\,]''. Each note is
+introduced by ``[\,\textit{Note:}'' and terminated by ``\textit{\,---\,end note}\,]''. Examples
```

Comment:
If we're not going to use the new environments here (which I agree is the right change) then it might be good to add a comment to the definitions of the environments in `macros.tex` saying something like "% if these change then make sure to update [intro.structure] to match"